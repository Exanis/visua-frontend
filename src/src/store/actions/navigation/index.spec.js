import { redirect } from './index';

describe('Navigation Actions', () => {
    it('Redirect: should redirect you', () => {
        expect(redirect('/test-location')).toMatchSnapshot();
    });
});