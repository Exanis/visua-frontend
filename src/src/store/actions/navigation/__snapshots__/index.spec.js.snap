// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navigation Actions Redirect: should redirect you 1`] = `
Object {
  "payload": Object {
    "args": Array [
      "/test-location",
    ],
    "method": "push",
  },
  "type": "@@router/CALL_HISTORY_METHOD",
}
`;
